Duplicati/Server/Duplicati.Server.csproj

```diff
@@ -168,6 +168,10 @@
       <Project>{F61679A9-E5DE-468A-B5A4-05F92D0143D2}</Project>
       <Name>Duplicati.Library.Backend.FTP</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Library\Backend\Idrivee2\Duplicati.Library.Backend.Idrivee2.csproj">
+      <Project>{6b594d23-b629-465c-b799-70ee9e56c218}</Project>
+      <Name>Duplicati.Library.Backend.Idrivee2</Name>
+    </ProjectReference>

```

Comment:
This needs to be added to the BackendTester, BackendTool and Duplicati.GUI.TrayIcon csproj files as well, or the installer build will not always pick them up.

Reply:
Done, Thanks @kenkendk . Really appreciated your time.